drivers/soundwire/intel.c

```diff
@@ -610,6 +612,7 @@ static int intel_params_stream(struct sdw_intel *sdw,
 	params_data.link_id = link_id;
 	params_data.alh_stream_id = alh_stream_id;
 
+	sdw->hw_params = hw_params;
```

Comment:
@bardliao   why not move "sdw->hw_params = hw_params;"  to intel_hw_params, like

sdw->pdi = pdi;
sdw->hw_params = hw_params;

I know: intel_params_stream is invoked in intel_prepare, but this line of code do nothing for this case.

Reply:
@RanderWang Good suggestion. Thanks.